src/content/docs/en/guides/astro-db.mdx

```diff
@@ -54,6 +54,7 @@ export default defineDb({
 Data in Astro DB is stored using SQL tables. Tables structure your data into rows and columns, where columns enforce the type of each row value. Astro DB supports the following column types:
 
 ```ts title="db/config.ts" "column.text()" "column.number()" "column.boolean()" "column.date()" "column.json()"
+import { defineTable, column } from 'astro:db';
```

Comment:
I thought readability could be improved by clearly indicating where `defineTable` and `column` are imported from.